README.md

```diff
@@ -1,2 +1,159 @@
+      }`;
+
+   int value = 1;
+   byte[] bytes = check avroSerDes.serialize(schema, value, "subject");
+   int number = check avroSerDesClient.deserialize(bytes);
+}
+```
+
+### Step 4: Run the Ballerina application
+
+Use the following command to compile and run the Ballerina program.
+
+```bash
+bal run
+```
+
+## Issues and projects
+
+The **Issues** and **Projects** tabs are disabled for this repository as this is part of the Ballerina library. To report bugs, request new features, start new discussions, view project boards, etc., visit the Ballerina library [parent repository](https://github.com/ballerina-platform/ballerina-library).
+
+This repository only contains the source code for the package.
+
+## Building from the source
+
+### Prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 17. You can download it from either of the following sources:
+
+   - [Oracle JDK](https://www.oracle.com/java/technologies/downloads/)
+   - [OpenJDK](https://adoptium.net/)
+
+    > **Note:** After installation, remember to set the `JAVA_HOME` environment variable to the directory where JDK was installed.
+
+2. Download and install [Ballerina Swan Lake](https://ballerina.io/).
+
+3. Download and install [Docker](https://www.docker.com/get-started).
+
+    > **Note**: Ensure that the Docker daemon is running before executing any tests.
+
+4. Generate a Github access token with read package permissions, then set the following `env` variables:
+
+    ```bash
+   export packageUser=<Your GitHub Username>
+   export packagePAT=<GitHub Personal Access Token>
+    ```
+
+### Build options
+
+Execute the commands below to build from the source.
+
+1. To build the package:
+
+   ```bash
+   ./gradlew clean build
+   ```
+
+2. To run the tests:
+
+   ```bash
+   ./gradlew clean test
+   ```
+
+3. To build the without the tests:
+
+   ```bash
+   ./gradlew clean build -x test
+   ```
+
+4. To debug package with a remote debugger:
+
+   ```bash
+   ./gradlew clean build -Pdebug=<port>
+   ```
+
+5. To debug with Ballerina language:
+
+   ```bash
+   ./gradlew clean build -PbalJavaDebug=<port>
+   ```
+
+6. Publish the generated artifacts to the local Ballerina central repository:
+
+   ```bash
+   ./gradlew clean build -PpublishToLocalCentral=true
+   ```
+
+7. Publish the generated artifacts to the Ballerina central repository:
+
+   ```bash
+   ./gradlew clean build -PpublishToCentral=true
+   ```
+
+## Contributing to Ballerina
+
+As an open source project, Ballerina welcomes contributions from the community.
+
+For more information, go to the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md).
+
+## Code of conduct
+
+All contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
+
+## Useful links
+
+- Discuss code changes of the Ballerina project in [ballerina-dev@googlegroups.com](mailto:ballerina-dev@googlegroups.com).
```

Comment:
API doc link is missing here right ?

Reply:
Added in [0fca32b](https://github.com/ballerina-platform/module-ballerinax-confluent.cavroserdes/pull/1/commits/0fca32b45cc1e6ccca077069fa1d9b01255ed71b)